Serialiser_Engine/Convert/OldJson.cs

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
How necessary is this file for the ongoing strategy/plan?

Reply:
If we intent to keep it, we should put some deprecation warnings on it to avoid people using it in the long run and sleep walking into problems?

Reply:
long terms strategy for this file is for it to be removed. Think that can be done in a separate PR though.

The risk of people using this I would say is quite low, given that most calls to the serialiser go via some mechnism that will point at the new system, but guess we could raise warnings if it is called or similar.

But from my point of view, I think it is fairly safe to keep it in for now, and then remove in a separate PR.